docs/contribute.rst

```diff
@@ -34,17 +34,18 @@ label. Those tickets are meant to be standalone and can be worked on ad-hoc.
 When contributing code, then please follow the standard Contribution
 Guidelines set forth at `contribution-guide.org`_.
 
-We have a strict code style that it's easy to follow since you just
-have to run a couple of commands and they will do everything for
-you. These commands are a mix between `autoflake`_, `autopep8`_,
-`docformatter`_, `isort`_, `unify`_ and `yapf`_::
-
-  $ autoflake --remove-all-unused-imports --remove-unused-variables --keep-useless-pass
-  $ autopep8
-  $ docformatter --wrap-summaries=80 --wrap-descriptions=80 --pre-summary-newline --no-blank
-  $ isort
-  $ unify --quote="'"
-  $ yapf --exclude=*migrations* --exclude=*settings* --exclude=*scripts* --parallel
+We have a strict code style that it's easy to follow since you just have to
+install `pre-commit`_ and it will be ran automatically as a pre-commit hook for
+git. This hook will run a different linting tools (`autoflake`_, `autopep8`_,
+`docformatter`_, `isort`_, `prospector`_, `unify`_ and `yapf`_) to check your
+changes before you commit them and let you know if there are some problems that
+weren't able to fix automatically.
+
+To install the `pre-commit` tool and setup the git pre-commit hook run these
+commands::
+
+  $ pip install -U pre-commit
+  $ pre-commit install
```

Comment:
Perhaps note the pre-commit install step as a separate "beta" usage of precommit hooks